tez-ui/pom.xml

```diff
@@ -29,7 +29,7 @@
   <properties>
     <webappDir>src/main/webapp</webappDir>
 
-    <nodeVersion>v5.12.0</nodeVersion>
+    <nodeVersion>v8.9.0</nodeVersion>
```

Comment:
maybe we can try with v16 ?
https://nodejs.org/en/about/releases/

Reply:
@slachiewicz When I update the node version to 16.15.1, this is the error that I get :
yarn run v1.6.0
(node:2298) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
$ TMPDIR=tmp node/node ./node_modules/ember-cli/bin/ember build -prod

exports.dir               = path.resolve(os.tmpDir());
                                            ^

TypeError: os.tmpDir is not a function
    at Object.<anonymous> (/home/tez-ui/src/main/webapp/node_modules/temp/lib/temp.js:273:45)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/amanraj2520/Repositories/wildfire-tez/tez-ui/src/main/webapp/node_modules/ember-cli/lib/tasks/install-blueprint.js:7:21)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Reply:
What I think this error means is that we need to replace all the places of occurrence of new Buffer() to either Buffer.alloc() or Buffer.from(). Let me know you thoughts

Reply:
Probably stack should be upgraded. I commented on Node's version because currently 16 is LTS version. I don't have experience with frontend tools.

Reply:
Yes thats true, but I think it will be another big change. I think we should make this change first and then work on upgrading the stack. Any thoughts ? @abstractdog @slachiewicz